test/verify/check-packagekit

```diff
@@ -920,42 +928,43 @@ class TestAutoUpdates(PackageCase):
         self.assertFalse(b.is_present("#auto-update-type"))
         assertTimer(None)
 
-        # OnCalendar= parsing: only time
-        m.execute("mkdir -p /etc/systemd/system/dnf-automatic.timer.d")
-        m.execute(r'printf "[Timer]\nOnUnitInactiveSec=\nOnCalendar=08:00\n" > '
-                  r'/etc/systemd/system/dnf-automatic.timer.d/cal.conf; systemctl daemon-reload')
-        b.reload()
-        b.enter_page("/updates")
-        b.click("#automatic .onoff-ct input")
-        b.wait_present("#automatic .onoff-ct input:checked")
-        b.wait_in_text("#auto-update-time", "08:00")
-        b.wait_in_text("#auto-update-day", "every day")
-
-        # OnCalendar= parsing: weekday and time
-        m.execute(r'printf "[Timer]\nOnUnitInactiveSec=\nOnCalendar=Tue 20:00\n" > '
-                  r'/etc/systemd/system/dnf-automatic.timer.d/cal.conf; systemctl daemon-reload')
-        b.reload()
-        b.enter_page("/updates")
-        b.wait_present("#automatic .onoff-ct input:checked")
-        b.wait_in_text("#auto-update-time", "20:00")
-        b.wait_in_text("#auto-update-day", "Tuesday")
-
-        # OnCalendar= parsing: "every day" calendar and time
-        m.execute(r'printf "[Timer]\nOnUnitInactiveSec=\nOnCalendar=*-*-* 07:00\n" > '
-                  r'/etc/systemd/system/dnf-automatic.timer.d/cal.conf; systemctl daemon-reload')
-        b.reload()
-        b.enter_page("/updates")
-        b.wait_present("#automatic .onoff-ct input:checked")
-        b.wait_in_text("#auto-update-time", "07:00")
-        b.wait_in_text("#auto-update-day", "every day")
-
-        # OnCalendar= parsing: unsupported
-        m.execute(r'printf "[Timer]\nOnUnitInactiveSec=\nOnCalendar=*-02-* 11:00\n" > '
-                  r'/etc/systemd/system/dnf-automatic.timer.d/cal.conf; systemctl daemon-reload')
-        b.reload()
-        b.enter_page("/updates")
-        b.wait_in_text(".container-fluid div.blank-slate-pf", "up to date")
-        b.wait_not_present("#automatic")
+        if self.backend == "dnf":
```

Comment:
Is this needed? I think the test will return if the backend isn't supported (e.i. dnf) no?

Reply:
It's not needed right now. The idea was that this prevents breaking the test once we add an automatic backend for apt. It's not important, it just caught my eye when I went through the tests. I can back it out if you prefer.